aspnetcore/security/app-secrets.md

```diff
@@ -31,53 +31,43 @@ For example, if you create a new ASP.NET Core web app with individual user accou
 
 ## Secret Manager
 
-The Secret Manager tool provides a more general mechanism to store sensitive data for development work outside of your project tree. The Secret Manager tool is a project tool that can be used to store secrets for a [.NET Core](https://microsoft.com/net/core) project during development. With the Secret Manager tool you can associate app secrets with a specific project and share them across multiple projects.
+The Secret Manager tool provides a general mechanism to store sensitive data for development work outside of your project tree. The Secret Manager tool is a project tool that can be used to store secrets for a [.NET Core](https://microsoft.com/net/core) project during development. With the Secret Manager tool you can associate app secrets with a specific project and share them across multiple projects.
 
 >[!WARNING]
 > The Secret Manager tool does not encrypt the stored secrets and should not be treated as a trusted store. It is for development purposes only. The keys and values are stored in a JSON configuration file in the user profile directory.
 
-### Installing the Secret Manager tool
+### Installing the Secret Manager tool 
 
-* Add `Microsoft.Extensions.SecretManager.Tools` to the `tools` section of the *project.json* file and run `dotnet restore`. 
+Add `Microsoft.Extensions.SecretManager.Tools` tools to the *.csproj* file and run 
+`dotnet restore`. 
 
-* Test the Secret Manager tool by running the following command:
+[!code-xml[Main](app-secrets/sample/UserSecrets/csproj.xml?highlight=17)]
```

Comment:
I prefer to show actual values, even if it will eventually get out of date.  It's actually a good thing to be showing the version a sample was created with/for.  When you put {Version} you're not showing a sample of an actual value and you can't draw it from a working project.  And I expect it will be confusing for people to find a csproj.xml in the sample project.